src/Frontend/Core/Engine/Navigation.php

```diff
@@ -642,7 +597,7 @@ public static function setExcludedPageIds($pageIds)
 
         // go trough the page ids to add them to the excluded page ids for later usage
         foreach ($pageIds as $pageId) {
-            array_push(self::$excludedPageIds, $pageId);
+            self::$excludedPageIds[] = $pageId;
```

Comment:
If you use array_push() to add one element to the array it's better to use $array[] = because in that way there is no overhead of calling a function.